### DISCUSSION TITLE
Can I use existing graphviz format graph as the graph for graphRAG?

### DISCUSSION THREAD

**Discussion by ltroin** [microsoft/graphrag#846] [Q&A]
I want to use an existing Graphviz format relational graph, I have searched the discussions and issues but found no answer.

Could you please let me know if it is possible...? Thank you so much in advance.

**Comment by natoverse** [comment]
At the moment there's no easy way to integrate other graph formats. Our graph information is stored in the create_final_nodes.parquet and create_final_relationships.parquet files. It's possible you could script something up to map into those files and then use `graphrag.index --resume` command to continue indexing from there.
